frameworks/C/h2o/src/main.c

```diff
@@ -177,9 +183,7 @@ static int initialize_global_data(const config_t *config, global_data_t *global_
 	sigset_t signals;
 
 	memset(global_data, 0, sizeof(*global_data));
-	global_data->config = config;
 	global_data->memory_alignment = get_maximum_cache_line_size();
```

Comment:
Cache line size relates to CPU caches, not HTTP request, database or any other higher-level caching mechanism, and yes, it is used as a memory management parameter.